man/pivot_wider_spec.Rd

```diff
@@ -25,6 +25,14 @@ build_wider_spec(
 \arguments{
 \item{data}{A data frame to pivot.}
 
+\item{spec}{A specification data frame. This is useful for more complex
```

Comment:
Hmmmm, I'm not 100% sure this is correct — I think it needs to be flipped around for `pivot_wider()`. ie. it gives you greater control on how metadata stored in the columns become column names. I think the best approach would be to just copy and paste the docs from `build_longer_spec()` and modify as appropriate.